extensions/binding/org.eclipse.smarthome.binding.mqttbroker/README.md

```diff
@@ -0,0 +1,82 @@
+# MqttBroker Binding
+
+    MQTT is a machine-to-machine (M2M)/"Internet of Things" connectivity protocol. It was designed as an extremely lightweight publish/subscribe messaging transport.
+
+This binding allows to list textual configured Mqtt brokers for connection status check purposes as well as configure new broker connections. The hereby setup broker connections are shared with the Eclipse Smarthome framework and can be enumerated and consumed by other bindings.
+
+## Supported Things
```

Comment:
I assume this has been already explained somewhere, but please can you summarize why not using a bridge?
Also if there is currently not a special bridge specific code wouldn't it make sense?

Reply:
This is just a decision that was made. Why does it need to be a bridge? I could also argue that the MQTT service should be a Thing and found connections should be dynamically added channels.

Reply:
@davidgraeff As you touched the whole MQTT related stuff I would like to understand why you have choosen a Thing but not a bridge.
It is not about that I would like to argue you have to do so, ...
I assume the decision has some background and not only a feeling.

If we create some time things with channel using MQTT topics, wouldn't it make sense that a broker the topics are received from acts like a bridge?
Or do you consider to allow create things from topics of different brokers?

Reply:
Those two bundles act independently. ESH Things are to be used when in doubt, so for this specific bundle the connections are modeled as Things. There is nothing supervising the connections (like a bridge) and there is nothing managed (like channels).

Other bundles can use the connections to make something useful with it. An extension could for example listen to a topic "super/important/notification" no matter of the exact broker. Another extension can implement a MQTT IoT convention with bridges/things/channels.